docs/README.md

```diff
@@ -1,20 +1,24 @@
 # Overview
 ![k0s logo](img/k0s-logo-full-color.svg)
 
-k0s is an all-inclusive Kubernetes distribution with all the required bells and whistles preconfigured to make building a Kubernetes clusters a matter of just copying an executable to every host and running it.
+k0s is an all-inclusive Kubernetes distribution, configured with all of the features needed to build a Kubernetes cluster simply by copying and running an executable file on each target host.
 
 ## Key Features
-- Packaged as a single static binary
-- Self-hosted, isolated control plane
-- Variety of storage backends: etcd, SQLite, MySQL (or any compatible), PostgreSQL
-- Elastic control-plane
+- Available as a single static binary
+- Offers a self-hosted, isolated control plane
+- Supports a variety of storage backends, including etcd, SQLite, MySQL (or any
+  compatible), and PostgreSQL.
+- Offers an Elastic control plane
 - Vanilla upstream Kubernetes
 - Supports custom container runtimes (containerd is the default)
-- Supports custom Container Network Interface (CNI) plugins (calico is the default)
+- Supports custom Container Network Interface (CNI) plugins (calico is the
+  default)
 - Supports x86_64 and arm64
 
 ## Join the Community
-If you'd like to help build k0s, please check out our guide to [Contributing](contributors/overview.md) and our [Code of Conduct](contributors/CODE_OF_CONDUCT.md).
+We welcome your help in continuing the development of k0s! If you are
```

Comment:
Somehow this sounds like we are giving up on the development and want someone to continue it.

Reply:
Changed the wording to "We welcome your help in building k0s!" (though, to be honest, I do not see much difference.)